bacon-vs-kefir-api.md

```diff
@@ -155,11 +155,11 @@
 | ----- | ----- | -------- |
 | `stream.map(property)` | Use `.sampledBy()` |  |
 | `obs.filter(property)` | `obs.filterBy(obs)` |  |
-| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Only property supported in Bacon? |
+| `obs.takeWhile(property)` | `obs.takeWhileBy(obs)` | Bacon supports property or predicate, Kefir supports any observable |
 | `obs.combine(obs2, fn)` | `obs.combine(obs2, [fn])` | In Kefir `fn` optional |
-| `stream.skipWhile(property)` | `obs.skipWhileBy(otherObs)` | Only property supported in Bacon? |
-| `stream.skipUntil(stream2)` | `obs.skipUntilBy(otherObs)` | Only stream supported in Bacon? |
-| `obs.takeUntil(stream) ` | `obs.takeUntilBy(otherObs)` | Only stream supported in Bacon? |
+| `stream.skipWhile(property)` | `obs.skipWhileBy(otherObs)` | Bacon supports property or predicate, Kefir supports any observable |
+| `stream.skipUntil(stream2)` | `obs.skipUntilBy(otherObs)` |  |
+| `obs.takeUntil(stream) ` | `obs.takeUntilBy(otherObs)` |  |
```

Comment:
Does that mean that `.skipUntil` and `.takeUntil` supports any observable in both positions? We should fix the signatures then too. Of if signatures correct, there should be a note.